sdk/mobile-center-analytics/build.gradle

```diff
@@ -1,6 +1,3 @@
-//
```

Comment:
In recent versions of Android Studio you cannot be confused anymore, the title of the editor window is now named after the module.